### ISSUE TITLE
Create section on open source contribution initiatives

### ISSUE THREAD

**Issue by erictleung** [freeCodeCamp/how-to-contribute-to-open-source#230]
Create a new section in the `README.md` labeled `Open source contribution initiatives`.

```
## Open source contribution initiatives
```

And add this to the table of contents.

```
- [Open source contribution intiatives](#open-source-contribution-initiatives)
```

Some of the content that can go in there that already exist in the README are:

- Up for Grabs
- First Timers Only
- First Contributions

And this can be added new as well:

- [Hacktoberfest](https://hacktoberfest.digitalocean.com/)

**Comment by p-murli** [comment]
Hey, I've added the following changes and created a PR. Could you review it and check if its all good?

**Comment by erictleung** [comment]
Closed by #231
